modules/getting-started-web-console-deploying-first-image.adoc

```diff
@@ -1,32 +1,45 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-deploying-first-image_{context}"]
-= Deploying your first image
+= Deploying the front-end application
 
-The simplest way to deploy an application in {product-title} is to run an existing container image. The following procedure deploys a front end component of an application called `national-parks-app`. The web application displays an interactive map. The map displays the location of major national parks across the world.
+The simplest way to deploy an application in {product-title} is to run a provided container image.
+
+The following procedure deploys `parksmap`, which is the front-end component of the `national-parks-app` application. The web application displays an interactive map of the locations of national parks across the world.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
```

Comment:
@bergerhoffer We should keep both these pre-reqs, otherwise the user won't understand in which perspective should they be logged to execute the following steps.

Reply:
Ignoring per our slack convo, since 4.19 got rid of these perspectives